Testscripts/Linux/dpdkUtils.sh

```diff
@@ -329,7 +365,16 @@ function Install_Dpdk () {
 		check_exit_status "${1} CONFIG_RTE_LIBRTE_MLX4_PMD=y" "exit"
 		ssh "${1}" "sed -i 's/^CONFIG_RTE_LIBRTE_MLX5_PMD=n/CONFIG_RTE_LIBRTE_MLX5_PMD=y/g' $RTE_SDK/config/common_base"
 		check_exit_status "${1} CONFIG_RTE_LIBRTE_MLX5_PMD=y" "exit"
-		ssh "${1}" "cd $RTE_SDK && meson $RTE_TARGET"
+		if [[ ${DISTRO_NAME} == rhel ]] && ! [[ ${DISTRO_VERSION} == *"8."* ]]; then
+			ssh ${1} "cd ${LIS_HOME}/${DPDK_DIR} && PATH=$PATH:/opt/rh/rh-python36/root/usr/bin/ && meson ${RTE_TARGET}"
+		else
+			ssh ${1} "cd ${LIS_HOME}/${DPDK_DIR} && meson ${RTE_TARGET}"
+		fi
+		if [[ ${DISTRO_NAME} == rhel || ${DISTRO_NAME} == centos ]]; then
```

Comment:
Why does it duplicate? 354 and 355 are the same.

Reply:
fixed